install_deps.cmake

```
@@ -113,5 +124,10 @@ cget(init ${TOOLCHAIN_FLAG} -DCMAKE_INSTALL_RPATH=${PREFIX}/lib ${PARSE_UNPARSED
 cget(ignore pcre)
 
 # Install dependencies
-cget(install -U ROCm/rocm-recipes@92c6695449c85887962f45509b376f2eb0d284f7)
-cget(install -U -f requirements.txt)
+if(DEFINED GENERATOR)
+    cget(install -G ${GENERATOR} -U ROCm/rocm-recipes@92c6695449c85887962f45509b376f2eb0d284f7)
+    cget(install -G ${GENERATOR} -U -f requirements.txt)
+else()
+    cget(install -U ROCm/rocm-recipes@92c6695449c85887962f45509b376f2eb0d284f7)
+    cget(install -U -f requirements.txt)
+endif()
```

```suggestion
cget(install ${CGET_EXTRA_OPTIONS} -U ROCm/rocm-recipes@92c6695449c85887962f45509b376f2eb0d284f7)
cget(install ${CGET_EXTRA_OPTIONS} -U -f requirements.txt)
```

I suggest instead of having a dedicated generator value having an `CGET_EXTRA_OPTIONS` cmake variable. That would allow adding support for more options when needed without making a huge if-else tree or rewriting.